src/tiled/tileanimationeditor.cpp

```diff
@@ -270,12 +270,10 @@ TileAnimationEditor::TileAnimationEditor(QWidget *parent)
     mUi->setupUi(this);
     resize(Utils::dpiScaled(size()));
 
-    Zoomable *zoomable = new Zoomable(this);
-    zoomable->setComboBox(mUi->zoomComboBox);
-
     mUi->frameList->setModel(mFrameListModel);
     mUi->tilesetView->setMarkAnimatedTiles(false);
-    mUi->tilesetView->setZoomable(zoomable);
+    //TODO: test
```

Comment:
Ah, you can open it using _Tileset -> Tile Animation Editor_ (when you have a tileset open).